user_sync/app.py

```diff
@@ -340,6 +347,7 @@ def main():
     try:
         try:
             args = process_args()
+
```

Comment:
PEP8: no blank lines in a try/except sequence.  Also it's very bad practice in a PR to make changes to code that has nothing to do with the code you mean to fix, because it produces unnecessary merge errors.